datafusion/sqllogictest/test_files/array.slt

```diff
@@ -2454,6 +2454,113 @@ select list_has_all(make_array(1,2,3), make_array(4,5,6)),
 ----
 false true false true
 
+## array_except
+
+statement ok
```

Comment:
Unlike other tests, I try a new style of test, hope this is easier for review.

Reply:
this is choice of style, imho `no-table` tests are easier to read.
```
query ?
select array_expect([], []), array_except(....)
----
[], ...
```

Reply:
Well, we still need to test multi-rows cases. The new style I mean is not create table in the beginning of this large file and drop table at the end of the file, but when we are done we drop it.

Reply:
the multirow still possible to do without creating a table 

```
select ... from (select 1 a, 'asdf' b union all select 2 b, 'zxcv' b)
```

thats the matter of style of course.

Reply:
We cant differentiate empty array and null currently. So we can fix them in another PR

Reply:
I also prefer the new style (create/query/drop table statements are organized more tightly). This will make tests more clear.